packages/cli/src/commands/generate/sdl/templates/sdl.js.template

```diff
@@ -1,28 +1,63 @@
 export const schema = gql`
+  <% if (docs) { %>
+  """${modelDescription}"""
+  <% } %>
   type ${singularPascalName} {
     ${query}
   }
+
 <% if (enums.length > 0) {%>
-<% enums.forEach((enumDef, idx)=> {%>  enum ${enums[idx].name} {<% enums[idx].values.forEach((enumDefValue, idk)=> { %>
+<% enums.forEach((enumDef, idx)=> {
+  const enumName = enums[idx].name
+  const enumDescription = enums[idx].documentation || `Possible values for ${enums[idx].name}`
+  %>
+  <% if (docs) { %>
+  """${enumDescription}"""
+  <% } %>
+  enum ${enums[idx].name} {<% enums[idx].values.forEach((enumDefValue, idk)=> { %>
     ${enums[idx].values[idk].name}<% }) %>
   }
 <%}) %><% } %>
+  <% if (docs) { %>
+  """About queries"""
+  <% } %>
   type Query {
-    ${pluralCamelName}: [${singularPascalName}!]! @requireAuth<% if (crud) { %>
-    ${singularCamelName}(id: ${idType}!): ${singularPascalName} @requireAuth<% } %>
+    <% if (docs) { %>
+    """Fetch ${pluralPascalName}."""
+    <% } %>${pluralCamelName}: [${singularPascalName}!]! @requireAuth<% if (crud) { %>
+    <% if (docs) { %>
+    """
+    Fetch a ${singularPascalName} by id.
+    """
```

Comment:
```suggestion
    """Fetch a ${singularPascalName} by id."""
```

Reply:
@dthyresson you can make this change if you use one `"` instead of three:

```
"Fetch a ${singularPascalName} by id."
```

This is also how the GraphQL spec has it: http://spec.graphql.org/draft/#example-916f4.

Reply:
The spec says that Descriptions are "String Values" which are: http://spec.graphql.org/October2021/#StringValue. I won't pretend to understand the first one, but the other two are just one quote or three quotes.

Reply:
I made the changes in the js/ts templates. Prettier may change them when saving the sdl.